src/main/java/com/footbalmanager/app/controller/LeagueController.java

```diff
@@ -0,0 +1,45 @@
+package com.footbalmanager.app.controller;
+
+import com.footbalmanager.app.dto.league.PatchLeagueRequestDto;
+import com.footbalmanager.app.dto.league.PostLeagueRequestDto;
+import com.footbalmanager.app.services.LeagueService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@CrossOrigin("http://localhost:8080")
```

Comment:
nie chcemy tego wyrzucić gdzieś do wspólnego interfejsu, możemy zrobić naszą adnotacje @RestController która będzie to zawierała żeby tego nie powielać, no i ten adres url najlepiej żeby był w jakimś konfigu.